app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class RentsController < Api::ApiController
+      include Wor::Paginate
+
+      def index
+        rents = current_api_v1_user.rents
```

Comment:
if you do it like this it will load on memory all the rents for the user, try using the DB for this request ( `Rent.where(user_id: ....)`